client/content/tutorials/socially/angular2/tutorials.socially.angular2.step_00.md

```diff
@@ -155,7 +159,7 @@ Finally, we `bootstrap` our component, thus, marking it as the root component.
 
 > You can see another example of Meteor's power and simplicity - no need to include this file anywhere. Meteor takes care of it by going through all the files in the project and include them automatically.
 
-Our component is linked to it's template with the `templateUrl` property, let's create the template:
+Our component has the template defined with the `template` property, let's create the template:
```

Comment:
Are you sure that with this functions?....I create a new project ...but this doesn't work, just with templateUrl